scripts/globals/pets/wyvern.lua

```diff
@@ -133,13 +133,17 @@ function onMobSpawn(mob)
             local diff = math.floor((prev_exp + currentExp)/200) - math.floor(prev_exp/200)
             if diff ~= 0 then
                 -- wyvern levelled up (diff is the number of level ups)
-                pet:addMod(dsp.mod.ACC,2*diff)
-                pet:addMod(dsp.mod.HPP,5*diff)
+                pet:addMod(dsp.mod.ACC,6*diff)
+                pet:addMod(dsp.mod.HPP,6*diff)
                 pet:addMod(dsp.mod.ATTP,5*diff)
                 pet:setHP(pet:getMaxHP())
                 player:messageBasic(dsp.msg.basic.STATUS_INCREASED, 0, 0, pet);
+                master:addMod(dsp.mod.ATTP,4*diff)
+                master:addMod(dsp.mod.DEFP,4*diff)
+                master:addMod(dsp.mod.HASTE_ABILITY,20*diff)
             end
             pet:setLocalVar("wyvern_exp", prev_exp + exp)
+            mob:setLocalVar("level_Ups", diff)
```

Comment:
this needs to be `mob:setLocalVar("level_Ups", mob:getLocalVar("level_Ups") + diff)`, or else it's going to keep overwriting with the wrong number

diff is just the number of levels that the wyvern just got, which will usually be 1 (unless you killed something that gave  more than 1 level of exp)